wildfly-cloud-legacy-galleon-pack/src/main/resources/configs/standalone/standalone.xml/config.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" ?>
 
+<!-- The default config expects the wildfly-datasources-galleon-pack to be present in the provisioning config -->
```

Comment:
Does that mean that the wildfly-cloud-legacy-galleon-pack as a (non-optional) dependency on wildfly-datasources-galleon-pack.

Does that mean that https://github.com/wildfly/wildfly-s2i/pull/173/files#diff-f98913fa22922d628a0ea4d0523af9a716ce1b51c888b7a81d8ce3ea30b3b9e0R75-R80 is always required to ensure both feature packs are provisioned?

Reply:
There is no explicit dependency on wildfly-datasources-galleon-pack. In order to provision the default configuration a user would provision both wildfly-coud-legacy-galleon-pack and wildfly-datasources-galleon-pack.

The jaxws example provision both feature-packs but that is not related to the default config. Here we are provisioning only layers. You will notice that in the jaxws example we also provision postgresql-driver driver. It is actually not needed, I should remove it.